CHANGES.md

```diff
@@ -1,3 +1,58 @@
+## Version 0.14.0
+
+This release is a major milestone for Spire.
+
+First of all, it introduces a dependency on typelevel/algebra (and
+thus on typelevel/cats-kernel). This provides immediate compatibility
+with Cats and Algebird.
+
+Second of all, Spire is now published to the org.typelevel
+organization (which was previously org.spire-math).
+
+Additions:
+ * Add GCDRing, generalized from EuclideanRing
+ * Add DivisionRing, a field without commutative multiplication
+
+Changes:
+ * Many type classes are now aliases to algebra's type classes
+ * The relationship between EuclideanRing and Gcd has changed
```

Comment:
Compared to algebra `Field <: CRing`, we have the hierarchy `spire.algebra.Field <: EuclideanRing <: GCDRing <: CRing`, with `spire.algebra.Field` extending both `EuclideanRing` and `algebra.Field`.

I was waiting for the @tixxit microsite to document these changes; if we release, I should update the original GUIDE as well.